packages/admin-ui/src/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { makeDecoratable, withStaticProps } from "~/utils";
+import { DrawerContent } from "./components/DrawerContent";
+import { DrawerFooter } from "./components/DrawerFooter";
+import { DrawerHeader } from "./components/DrawerHeader";
+import { DrawerOverlay } from "./components/DrawerOverlay";
+import { DrawerPortal } from "./components/DrawerPortal";
+import { DrawerRoot } from "./components/DrawerRoot";
+import { DrawerTrigger } from "./components/DrawerTrigger";
+import { DrawerBody } from "~/Drawer/components/DrawerBody";
+import { Icon } from "./components/Icon";
+import { ConfirmButton } from "./components/ConfirmButton";
+import { CancelButton } from "./components/CancelButton";
+
+export interface DrawerProps
+    extends React.ComponentPropsWithoutRef<typeof DrawerRoot>,
+        Omit<React.ComponentPropsWithoutRef<typeof DrawerContent>, "title"> {
+    trigger?: React.ReactNode;
+    title?: React.ReactNode;
+    icon?: React.ReactElement;
+    modal?: boolean;
+    showCloseButton?: boolean;
+    bodyPadding?: boolean;
+    description?: React.ReactNode;
+    children: React.ReactNode;
+    actions?: React.ReactNode;
+    info?: React.ReactNode;
+    width?: React.CSSProperties["width"];
+    tabs?: boolean;
```

Comment:
This prop is not used in the code.

Reply:
LOL good catch. I have no idea how this ended up in here. :D